macOS/DuckDuckGo/SecureVault/View/PasswordManagementItemList.swift

```diff
@@ -535,3 +536,16 @@ private struct HideMenuIndicatorModifier: ViewModifier {
     }
 
 }
+
+private struct FlexibleButtonSizingModifier: ViewModifier {
+
+    func body(content: Content) -> some View {
+        if #available(macOS 26, *) {
+            content
+                .buttonSizing(.flexible)
+        } else {
+            content
+        }
```

Comment:
We'll want to find a way to keep these changes but disable them until we're ready to move to Xcode 26 on CI, some time after it's released. The issue is that even with the availability check, older versions of Xcode will fail to compile this.

Could be as simple as commenting it out before we merge this with a `TODO` comment, and we create a follow-up task to uncomment them as part of the Xcode 26 upgrade later.